config/config.c

```diff
@@ -45,6 +45,9 @@ Settings config = {
 
     /** Whether to load and show icons */
     .show_icons = FALSE,
+    
```

Comment:
Small styling issue: there are trailing white spaces on empty lines in the whole diff. We can remove them before merging if possible.

Reply:
I recommend running clang-format with default settings, that should in general be the style used.